src/constants/index.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+    serverConfig: {
+        port: 4200,
```

Comment:
Consider using environment variable for future deployment purposes `port: process.argv.PORT || 4200`